docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -44,19 +49,17 @@ The Sekoia.io Endpoint Agent uses the HTTPS protocol to send its events and has
     * https://api.sekoia.io/
 
 === "FRA2"
-    * https://fra2.app.sekoia.io/
+    * <https://fra2.app.sekoia.io/>
 
 === "MCO1"
-    * https://mco1.app.sekoia.io/
-
-
+    * <https://mco1.app.sekoia.io/>
 
 ## Installation
 
 ### Disclaimer
 
 !!! Warning
-	If you want to install this agent on a machine with an EDR in place, please keep in mind that most EDRs perform actions to detect malware or other types of threats. Those actions generate events that our agent collects. This may result in raising false positive alerts from our detection rules of effort levels 3 and 4. Customers need to fine-tune these rules to reduce the occurrence of false positives.
+ If you want to install this agent on a machine with an EDR in place, please keep in mind that most EDRs perform actions to detect malware or other types of threats. Those actions generate events that our agent collects. This may result in raising false positive alerts from our detection rules of effort levels 3 and 4. Customers need to fine-tune these rules to reduce the occurrence of false positives.
```

Comment:
Needed to be displayed inside the warning box

![image](https://github.com/SEKOIA-IO/documentation/assets/679995/260697c9-37f3-4a2e-9dde-04c045dc4d6e)

```suggestion
	 If you want to install this agent on a machine with an EDR in place, please keep in mind that most EDRs perform actions to detect malware or other types of threats. Those actions generate events that our agent collects. This may result in raising false positive alerts from our detection rules of effort levels 3 and 4. Customers need to fine-tune these rules to reduce the occurrence of false positives.
```